android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerDelegate.java

```diff
@@ -42,7 +42,7 @@ public class FilePickerDelegate implements PluginRegistry.ActivityResultListener
     private boolean loadDataToMemory = false;
     private String type;
     private boolean allowCompression = true;
```

Comment:
Is it really the `compressionQuality` that dictates whether we compress images?
The `private boolean allowCompression = true;` line seems like the culprit instead?

Like, if we have `allowCompression` set to false, then we shouldn't be looking at the `compressionQuality` at all.
Can we leave the compressionQuality as-is, but change the default `allowCompression` to false instead?